<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="OliveButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Olive</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MushroomButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mushroom</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CheeseButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Goat Cheese</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MeatButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Meat</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="SouceText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 239, 254);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;* and the sauce of your choice:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="OnionButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Onion</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CornButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Corn</string>
    </property>
   </widget>
   <widget class="QLabel" name="OliveCost">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>6 ₺</string>
    </property>
   </widget>
   <widget class="QLabel" name="MushroomCost">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>8 ₺</string>
    </property>
   </widget>
   <widget class="QLabel" name="GoatCost">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>8 ₺</string>
    </property>
   </widget>
   <widget class="QLabel" name="MeatCost">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>10 ₺</string>
    </property>
   </widget>
   <widget class="QLabel" name="OnionCost">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4 ₺</string>
    </property>
   </widget>
   <widget class="QLabel" name="CornCost">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5 ₺</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
    <widget class="QLabel" name="DominosCost">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>20 ₺</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="BaseText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Please Select a Pizza Base:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="DominosButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Dominos</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="MargheritaButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Margherita</string>
     </property>
    </widget>
    <widget class="QLabel" name="MargerithaCost">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>25 ₺</string>
     </property>
    </widget>
    <widget class="QLabel" name="ClassicCost">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>20 ₺</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ClassicButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Classic</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="TurkishButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Turkish</string>
     </property>
    </widget>
    <widget class="QLabel" name="TurkishCost">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>30 ₺</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="PayButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>580</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PAY</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>1041</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/Just.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1041</width>
       <height>731</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/global_ai_python_project/Just.jpg);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>420</y>
        <width>1021</width>
        <height>741</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/new/Desktop/about.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>OliveButton</zorder>
   <zorder>MushroomButton</zorder>
   <zorder>CheeseButton</zorder>
   <zorder>MeatButton</zorder>
   <zorder>SouceText</zorder>
   <zorder>OnionButton</zorder>
   <zorder>CornButton</zorder>
   <zorder>OliveCost</zorder>
   <zorder>MushroomCost</zorder>
   <zorder>GoatCost</zorder>
   <zorder>MeatCost</zorder>
   <zorder>OnionCost</zorder>
   <zorder>CornCost</zorder>
   <zorder>widget</zorder>
   <zorder>PayButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../test.qrc"/>
 </resources>
 <connections/>
</ui>
